descriptors/ModuleDescriptor-template.json

```diff
@@ -69,7 +69,7 @@
     },
     {
       "id": "circulation",
-      "version": "12.0"
+      "version": "9.5 10.0 11.0 12.0"
```

Comment:
Why has dual support for multiple major versions of the interface been reintroduced when it was previously removed?

Reply:
As release build can not find new 12 version on dependency check  step - the previous versions were added

Reply:
Are you confident that the module still supports all of the interface versions? 

I'm assuming that the dual support was removed deliberately in a previous change. Have you checked with the folks who made that change that the dual support could have been retained?

Reply:
A quick check of the [pull request](https://github.com/folio-org/mod-patron/pull/97/files) where those changes were made suggests that the mod-patron code now relies on the structure of a request following the breaking changes made for title level requests e.g. the `createCancelRequest` or `createRequestByItem` methods which includes properties not in the old representation of requests.

I think that means this module cannot support interface versions before and after that change with the current code.

@alexanderkurash  @Mykyta-Varenyk Is my interpretation of that pull request wrong?

Reply:
@marcjohnson-kint Yes, that's correct, after the new changes from [this pr](https://github.com/folio-org/mod-patron/pull/97/files) this module is only compatible with 12+ circulation interface.

Reply:
@Mykyta-Varenyk Thank you for confirming my understanding.

@alexanderkurash @Roman-Leshchenko 

I think that means that the release done yesterday is invalid and misleading (in that it states it is compatible with versions of circulation that it is not).

I think a new release needs preparing that undoes that incorrect compatibility statement and the mainline needs updating to reflect that both of those releases have gone out (in order to unblock the hosted reference environments)

@craigmcnally  I think you may need to review how urgent releasing this feature is, given that a mainline release now won't work with pre-2022 R1 distributions.

Reply:
@marcjohnson-kint  reverted to 12.0 after disscussion with @alexanderkurash

Reply:
@Roman-Leshchenko 

> reverted to 12.0 after disscussion with @alexanderkurash

Ok. What is the intention behind reverting the change in this pull request? It won't help resolve the challenge with the release being misleading and inappropriate for use.

Reply:
I'm now a bit confused. The 5.1.0 tag seems to only declare compatibility with 12.0. Does that mean this change was made to this branch _after_ the release was made (and cannot be finished because there is no module version that provides 12.0)?